trio/_core/_run.py

```diff
@@ -910,7 +958,9 @@ def _child_finished(self, task, outcome):
         self._check_nursery_closed()
 
     async def _nested_child_finished(self, nested_child_exc):
-        """Returns MultiError instance if there are pending exceptions."""
+        """
```

Comment:
This isn't full documentation for the function (nor does it need such, it's not public) so should probably be an ordinary `# comment`. Also the result might not be a MultiError instance if we're in loose mode for this nursery.